README.md

```diff
@@ -1,2 +1,11 @@
 # XebiaLabs-Appium-ReactNative
 Guidance on integrating Kobiton service into the mobile app build pipeline: XebiaLabs, Appium and ReactNative.
```

Comment:
You can remove this line, I think the meaning of it is duplicating with "The guidelines in this repo will show you how to plan a release with the XebiaLabs service XL release as well as how to trigger a Jenkins build from a task on XebiaLabs."